From: Jan Kloetzke <jan@kloetzke.net>
Subject: Remove libmount from sysroot

Gcc won't pick up this library. If needed, it will need to be folded into libc
on the L4Re side.

--- a/Makefile	2025-05-23 08:04:52.199460490 +0200
+++ b/Makefile	2025-05-23 08:04:57.567698895 +0200
@@ -289,7 +289,7 @@
 
 # Build a typical sysroot for use with external tooling such as a
 # L4Re-specific cross-compiler
-SYSROOT_LIBS = libgcc libgcc_eh libgcc_s libc libpthread librt libdl libld-l4 libm libc_nonshared.p libmount
+SYSROOT_LIBS = libgcc libgcc_eh libgcc_s libc libpthread librt libdl libld-l4 libm libc_nonshared.p
 OUTPUT_FORMAT = $(CC) $(CFLAGS) -Wl,--verbose 2>&1 | $(SED) -n '/OUTPUT_FORMAT/,/)/p'
 sysroot: $(foreach p,ldso libc_backends uclibc,pkg/l4re-core/$(p))
 	$(GEN_MESSAGE)
